mnefun/_sss.py

```diff
@@ -1024,3 +1106,20 @@ def annotate_head_pos(raw, head_pos, rotation_limit=45, translation_limit=0.1,
 
     # Annotate on distance from the sensors
     return annot
+
+
+def check_sws():
+    """Check if SSS workstation is configured correctly."""
+    from . import Params
+    p = Params()  # loads config
```

Comment:
I also noticed, even after getting everything else set up right, the first call to ```Params()``` to create an instance of 'params' always leads to the same warning: "UserWarning: Configuration value p.sws_dir was None, remote Maxfilter processing will fail."  etc for the other sws parameters.  Nevertheless, examination of the resulting 'params' shows that those parameters are correctly populated from the config file.

Reply:
Whoops, this was a bug and should be fixed now